Code/Editor/Core/LevelEditorMenuHandler.cpp

```diff
@@ -723,7 +724,8 @@ QMenu* LevelEditorMenuHandler::CreateViewMenu()
     // MISSING AVIRECORDER
 
     viewportViewsMenuWrapper.AddSeparator();
-    viewportViewsMenuWrapper.AddAction(ID_DISPLAY_SHOWHELPERS);
+    viewportViewsMenuWrapper.AddAction(AzToolsFramework::Helpers);
+    viewportViewsMenuWrapper.AddAction(AzToolsFramework::Icons);
```

Comment:
I've got rid of another of one of the annoying old cry resource ids... it's now possible to register actions using reverse-urls which are much preferred. This moves us away from some more legacy cruft...